app/views/admin/tasks/_task_outcome.html.erb

```diff
@@ -3,6 +3,10 @@
     <%= task_status_tag(task.claim, task.name) %>
   </p>
   <p class="govuk-body">
-    This task was performed by <%= user_details(task.created_by, include_line_break: false) %> on <%= l(task.created_at) %>
+    <% if task.manual %>
+      This task was performed by <%= user_details(task.created_by, include_line_break: false) %> on <%= l(task.created_at) %>
+    <% else %>
+      This task was performed by an automated check on <%= l(task.created_at) %>
```

Comment:
I think we should still include the name of the user here so it's clear who did the automated check.